Evaluation/constants.tsx

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios';
+
+const API_URL = 'https://jsonplaceholder.typicode.com/posts';
+
+
+export interface Posts {
+  id: number;
+  title: string;
+  body: string; 
+}
+
+
+
+export const getPosts = () => axios.get<Posts[]>(API_URL);
```

Comment:
add the getPosts in a separate service folder.

Reply:
resolved all the comments